Loop/Managers/ServicesManager.swift

```diff
@@ -23,13 +23,16 @@ class ServicesManager {
 
     private let servicesLock = UnfairLock()
 
-    init(pluginManager: PluginManager, deviceDataManager: DeviceDataManager) {
+    init(
```

Comment:
If you are going to pass in both `remoteDataServicesManager` and `analyticsServicesManger` why not `loggingServicesManager`, too?